x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -123,6 +124,22 @@ export const getSavedObjectTypes = (
         },
       },
     },
+    [SPACE_SETTINGS_SAVED_OBJECT_TYPE]: {
+      name: SPACE_SETTINGS_SAVED_OBJECT_TYPE,
+      indexPattern: INGEST_SAVED_OBJECT_INDEX,
+      hidden: false,
+      namespaceType: 'single',
+      management: {
+        importableAndExportable: false,
+      },
+      mappings: {
+        dynamic: false,
+        properties: {
+          allowed_namespace_prefixes: { enabled: false },
+          managed_by: { type: 'keyword', index: false },
+        },
+      },
```

Comment:
If these fields are not `enabled` or `index`ed we should be able to exclude them from mappings definitions. In this case: `mappings: {}`.